pages/index.js

```diff
@@ -92,6 +94,19 @@ export default function Home({ locale, popularCategories, situationCookie }) {
     setBenefits(data);
   }, [situation]);
 
+  //only make location assumptions if the situation cookie doesn't have anything set for location
+  if (situation?.location === undefined) {
+    getUserLocationAssumption()
```

Comment:
Instead of setting the location assumption from index it would be easier when creating the cookie initially. Can we have a call to discuss options?